doc/install/install-kubernetes.md

```diff
@@ -1,7 +1,6 @@
 # Install Kuadrant on a Kubernetes cluster
 
 !!! note
-
```

Comment:
does this render ok with mkdocs (checkout a branch locally, update mkdocs.yaml to point to this branch for the kuadrant-operator component) ? could be wrong by I vaguely recall that maybe these newlines were needed - worth a quick check

Reply:
no good shout ill double check